testing/citest/tests/gcs_pubsub_gae_test.py

```diff
@@ -0,0 +1,449 @@
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Invoke this test by executing this python command:
+#
+# PYTHONPATH=$CITEST_ROOT \
+# python spinnaker/testing/citest/tests/gcs_pubsub_gae_test.py
+# --native_hostname
+# --native_port
```

Comment:
Add preconfig steps and remove parameters that arent defaults, adding default values where possible if they arent there already. Dont assume external values (e.g. not the name of the subscription, but teh app/stack). Ideally there is a public repo you can run this on, such as the one we use for appengine smoke test if this can be the same.

Reply:
Added.